KsApi/models/lenses/LocationLenses.swift

```diff
@@ -4,22 +4,32 @@ extension Location {
   public enum lens {
     public static let country = Lens<Location, String>(
       view: { $0.country },
-      set: { Location(country: $0, displayableName: $1.displayableName, id: $1.id, name: $1.name) }
+      set: { Location(country: $0, displayableName: $1.displayableName, id: $1.id,
```

Comment:
Thought we're replacing these with the 'new' `private(set) var` style :) Or are we just waiting out Swift 4's keypath stuff?